### ISSUE TITLE
trouble with rollup and json imports

### ISSUE THREAD

**Issue by micahscopes** [danigb/tonal#21]
`tonal-scale` seems to rely on rollup-plugin-json.  the only thing is that since I'm using rollup in my project, I had to figure out that I needed to use rollup-plugin-json in my rollup configuration too.  This seems like a funky area with rollup/es6 modules.

**Comment by danigb** [comment]
Didn't know... Any suggestion?

**Comment by micahscopes** [comment]
ahhh, I saw [here](https://github.com/danigb/tonal/blob/master/packages/scale/index.js#L20) that you're using `require` to include the `scales.json` file...  so you aren't relying on `rollup-plugin-json` after all.  I'm not sure why my rollup configuration is choking on that file without the plugin...

I need to dig deeper.  I just realized that I'd disabled jsnext in my build, which means I was using the commonjs version of tonal.  So I think that's why I needed that plugin.

But then when I went and re-enabled jsnext, I got an error with the `note-parser` module:
```
note-parser/index.js does not export parse
```
